ember-file-upload/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import UploadFile from './upload-file';
+import Queue from './queue';
+import HTTPRequest from './system/http-request';
+import DataTransferWrapper from './system/data-transfer-wrapper';
+import UploadFileReader from './system/upload-file-reader';
+import { FileSource, FileState } from './interfaces';
+import { DEFAULT_QUEUE } from './services/file-queue';
+
+export {
+  // Public API classes
+  Queue,
+  UploadFile,
+  // Undocumented classes
+  HTTPRequest,
+  DataTransferWrapper,
+  UploadFileReader,
```

Comment:
Any reasons to export these classes at all? If they are privat, maybe we don't need a public export from them.

Reply:
They're used in the test app. Otherwise I don't have a reason

Reply:
Maybe we could export them from `ember-file-upload/internal`? The import path would highlight that they are not part of public API. And editors won't include them as suggestion for autocompletion.

Reply:
Great suggestion – much more clarity. I have implemented this